script/functions.sh

```diff
@@ -28,7 +28,7 @@ function run_specs_and_record_done {
 
   # rspec-core needs to run with a special script that loads simplecov first,
   # so that it can instrument rspec-core's code before rspec-core has been loaded.
-  if [ -f script/rspec_with_simplecov ]; then
+  if [ -f script/rspec_with_simplecov ] && is_mri; then
```

Comment:
I'm not sure we've ever tried to run it before, our other JRuby modes report themselves as `< 2` so miss this anyway, but `rspec-core` is reporting 0% coverage on JRuby 9.1.50